python/sklearnserver/sklearnserver/model.py

```diff
@@ -28,8 +30,15 @@ def __init__(self, name: str, model_dir: str):
         self.ready = False
 
     def load(self):
-        model_file = os.path.join(kfserving.Storage.download(self.model_dir), JOBLIB_FILE) #pylint:disable=c-extension-no-member
-        self._joblib = joblib.load(model_file) #pylint:disable=attribute-defined-outside-init
+        model_path = kfserving.Storage.download(self.model_dir)
+        joblib_path = os.path.join(model_path, JOBLIB_FILE)
+        pickle_path = os.path.join(model_path, PICKEL_FILE)
+        if os.path.exists(joblib_path):
+            self._model = joblib.load(joblib_path) #pylint:disable=attribute-defined-outside-init
+        elif os.path.exists(pickle_path):
+            self._model = pickle.load(open(pickle_path, 'rb'))
```

Comment:
In my opinion, the joblib in fact support load pickle format files right? It's good we explicitly support pickle format but seems it's just file name issue, we can use joblib.load for both model.joblib and model.pkl

Reply:
Sorry I didn't know joblib can also load pickle formate files since most of the examples I found online are using pickle.load for pkl files. I will update the pickle files to also use joblib.load. Thanks.

Reply:
OK, it looks good to me now, thanks.